### DISCUSSION TITLE
是否有类似element的tag标签组件

### DISCUSSION THREAD

**Discussion by weboy9527** [layui/layui#2315] [建议]
是否有类似element的tag标签组件

**Comment by Sight-wcg** [comment]
https://layui.dev/docs/2/badge/
https://layui.dev/docs/2/form/checkbox.html#skin

layui 有没有 xx 组件？文档中有就是有，没有就是没有，不必重复提类似的问题。

**Comment by hulang** [comment]
https://dev.layuion.com/extend/search/tag/
